Gems/ROS2/Code/Source/Lidar/LidarRaycaster.h

```diff
@@ -50,7 +50,6 @@ namespace ROS2
         bool m_addMaxRangePoints{ false };
         AZStd::vector<AZ::Vector3> m_rayRotations{ { AZ::Vector3::CreateZero() } };
 
-        bool m_ignoreLayer{ false };
-        AZ::u32 m_ignoredLayerIndex{ 0 };
+        AZStd::vector<AZ::u32> m_ignoredLayerIndices{};
```

Comment:
Use unordered set instead, since as far as I understand you only need to either add to it or check whether element is present.

Reply:
I've thought a vector would be faster, since normally there's probably very few ignored layers

Reply:
However, it is true that a set would be clearer. Should I change it?